main.go

```diff
@@ -128,6 +129,8 @@ var supportedLocations = map[string]location{
 	"zurich": {nil, 47.369019, 8.538030, "Europe/Zurich", "Zurich"},
 	"berlin": {nil, 52.520008, 13.404954, "Europe/Berlin", "Berlin"},
 	"dubai":  {nil, 25.204849, 55.270782, "Asia/Dubai", "Dubai"},
+	"manila": {nil, 14.599512, 120.984222, "Asia/Dubai", "Manila"},
+	"hong kong": {nil, 22.317053, 114.169547, "Asia/Hong_Kong", "Hong Kong"},
```

Comment:
Keys should not contain space, as they are used in URLs. I suggest using `hk` to be consistent with other cities with similar abbreviation like `la` or `sf`.